pkg/fleetautoscalers/fleetautoscalers.go

```diff
@@ -31,6 +31,7 @@ import (
 	agonesv1 "agones.dev/agones/pkg/apis/agones/v1"
 	autoscalingv1 "agones.dev/agones/pkg/apis/autoscaling/v1"
 	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
```

Comment:
I think there is no need to have an alias for logrus:
```suggestion
	"github.com/sirupsen/logrus"
```